shared/query_engine/query_creator.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, session, project, member, directory = None):
         self.member = member
         self.directory = directory
         # Additional security check just for sanity
+        # TODO: This is duplicated from sqlalchemy_query_executor __init__, I think we can remove?
```

Comment:
I think this can be removed, exact same check is made in both the query creator and the query executor